roles/config-rh-sso/tasks/main.yml

```diff
@@ -17,3 +17,7 @@
   loop: "{{ roles }}"
   loop_control:
     loop_var: role_data
+- include_tasks: 'configure-id-provider.yml'
+  loop: "{{ id_providers }}"
```

Comment:
@BinaryDevotee Can you change this to `loop: "{{ id_providers | default([]) }}"` ? This will allow us to skip this if no id_providers are present.

Reply:
Fixed.